libs/blocks/aside/aside.css

```diff
@@ -137,11 +177,48 @@
   color: transparent;
 }
 
-.aside.split .split-image img {
+.aside.split .split-image img,
+.aside.split .split-image video {
   position: relative;
   min-height: 270px;
 }
 
+.aside.split .split-image video {
+  display: block;
+}
+
+.aside.split .mobile-square img,
+.aside.split .mobile-square video {
+  aspect-ratio: var(--aspect-ratio-square);
+}
+
+.aside.split .mobile-wide img,
+.aside.split .mobile-wide video {
+  aspect-ratio: var(--aspect-ratio-wide);
+}
+
+.aside.split .mobile-standard img,
+.aside.split .mobile-standard video {
+  aspect-ratio: var(--aspect-ratio-standard);
+}
+
+.aside.split .icon-stack-area li img {
+  height: 40px;
+  width: 40px;
+}
+
+.aside.split .format img,
+.aside.split .format video {
+  height: auto;
+}
+
+.aside.split .format picture,
+.aside.split .format video {
+  display: flex;
+  height: 100%;
+  align-items: center;
+}
+
 .aside .foreground.container .icon-area img {
   max-width: 234px;
   height: 56px;
```

Comment:
```suggestion
  height: var(--icon-size-xl);
```

Reply:
@ryanmparrish I have updated the icon sizes as suggested to a standard icon-size variable name.